app/src/main/java/io/homeassistant/companion/android/sensors/SensorsSettingsFragment.kt

```diff
@@ -96,6 +102,23 @@ class SensorsSettingsFragment : PreferenceFragmentCompat() {
 
         setPreferencesFromResource(R.xml.sensors, rootKey)
 
+        findPreference<SwitchPreference>("show_only_enabled_sensors")?.let {
+
+            it.summary = getShowOnlyEnabledSensorsSummary(getShowOnlyEnabledSensors())
```

Comment:
This can be done in the XML via this: https://developer.android.com/reference/androidx/preference/SwitchPreference#setSwitchTextOff(int)

Reply:
Thanks i used:  `android:summaryOff` /  `android:summaryOn`